src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/HttpInListener.cs

```diff
@@ -39,6 +37,8 @@ internal class HttpInListener : ListenerHandler
 
     private const string DiagnosticSourceName = "Microsoft.AspNetCore";
 
+    private static readonly bool Net7OrGreater = Environment.Version.Major >= 7;
```

Comment:
Let's be consistent and also update this preprocessor here
https://github.com/open-telemetry/opentelemetry-dotnet/blob/edfea52f2e2021c4cdcb174499efe8207e3a52f2/src/OpenTelemetry.Instrumentation.AspNetCore/AspNetCoreInstrumentationTracerProviderBuilderExtensions.cs#L104-L120

Reply:
04f79631a278b636648199f88164b28784a7f1bf